src/dlgpreflibrary.cpp

```diff
@@ -59,7 +59,7 @@ DlgPrefLibrary::DlgPrefLibrary(QWidget * parent,
 
     // plugins are loaded in src/main.cpp way early in boot so this is safe
     // here, doesn't need done at every slotUpdate
-    QStringList plugins(SoundSourceProxy::supportedFileExtensionsByPlugins());
+    QStringList plugins(SoundSourceProxy::getSupportedFileTypesByPlugins());
```

Comment:
I liked the old version more. We just check for the file extension, not the mime type or something from libmagic. 
Inside Mixxx we use inconsistent getter style some have get prefix some not. Qt does not use the get prefix ,,, just a matter of taste.